lib/migrations/migration_runner.js

```diff
@@ -260,7 +260,7 @@ var MigrationRunner = function () {
 
                 migration = _step3.value;
                 _context2.next = 22;
-                return migration.upgrade();
+                return migration.upgrade(this._server);
```

Comment:
Hmm maybe we should do it at configuration phase 
https://github.com/sirensolutions/kibiutils/blob/dc835196c420e63fdbe7a4a1d71ea1623d05e6b3/src/migrations/migration_runner.js#L52
There we are already passing config 
We could pass the whole server 

Then we probably would have to update all migrations so they grab the config themselves

Reply:
What if we just add the server object instead of replacing the config one? Would be retro-compatible with all the current migrations

Reply:
To avoid upgrading all migrations we could do 
```
const configuration = {
            config: this._server.config(),
            client: this._server.plugins.elasticsearch.getCluster("admin").getClient(),
            logger: this._logger,
            server: this_server   // <---- just add it here 
          };
```

Reply:
exactly 👍